src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -453,5 +453,15 @@ protected override ISymbol CreateISymbol()
         {
             return new PublicModel.FieldSymbol(this);
         }
+
+        public override bool Equals(Symbol other, TypeCompareKind compareKind)
```

Comment:
>Equals [](start = 29, length = 6)

If we override Equals, I would expect us to override GetHashCode as well.

Reply:
It will just call down into the base though, so it seems redundant?

Reply:
> It will just call down into the base though, so it seems redundant?

That will be informative for the reader and then future changes to Equals might require changes to GetHashCode too, which is easy to forget unless the methods next to each other.

---
In reply to: [370320715](https://github.com/dotnet/roslyn/pull/41123#discussion_r370320715) [](ancestors = 370320715)